packages/core/cli.js

```diff
@@ -52,6 +52,12 @@ if (userWantsGeneralHelp) {
   process.exit(0);
 }
 
+//if the last word of inputString is `help` or `--help`, re-assign inputString to run as `truffle help <cmd>`
+if (["help", "--help"].includes(inputStrings[inputStrings.length - 1])) {
```

Comment:
I think this logic should expand to include `truffle --help <cmd>` as it is a common way to request help for the specific command. I often hit this and then remember that truffle is different (context switch). This is an opportunity to be more forgiving. :)

- [✗] `truffle --help db` 
- [✓] `truffle help db`
- [✓] `truffle db help`
- [✓] `truffle db --help`

Reply:
I agree with @cds-amal on this. It is probably better to give more options to the user.

Reply:
I dig it. It was an overlook of my todos. Let me add this.